torch/fx/graph_module.py

```diff
@@ -194,10 +194,20 @@ def __reduce__(self):
     def __deepcopy__(self, memo):
         fake_mod = torch.nn.Module()
         fake_mod.__dict__ = copy.deepcopy(self.__dict__)
-        return GraphModule(fake_mod, self.graph)
+        graph_module = GraphModule(fake_mod, self.graph)
```

Comment:
these are changes from https://github.com/pytorch/pytorch/pull/45182, will rebase on master after it is landed.